src/aura/STG_CMP_Tabs/STG_CMP_Tabs.cmp

```diff
@@ -32,7 +35,8 @@
              <c:STG_CMP_Addr hierarchySettings="{!v.hierarchySettings}" isView="{!v.isView}" namespacePrefix="{!v.namespacePrefix}" />
          </div>
          <div aura:id="systemTabContent" class="site-code--content slds-scrollable--x slds-tabs__content" role="tabpanel">
-             <c:STG_CMP_System hierarchySettings="{!v.hierarchySettings}" isView="{!v.isView}" namespacePrefix="{!v.namespacePrefix}" />
+             <c:STG_CMP_System hierarchySettings="{!v.hierarchySettings}" accRecTypes="{!v.accRecTypes}" accRecTypeId="{!v.accRecTypeId}" 
+             accRecTypeName="{!v.accRecTypeName}" isView="{!v.isView}" namespacePrefix="{!v.namespacePrefix}" />
```

Comment:
The main reason is that you need the namespace when displaying labels, like in https://github.com/SalesforceFoundation/HEDAP/blob/feature/216-acc-rec-types-change/src/aura/STG_CMP_System/STG_CMP_System.cmp#L27. There is no way to choose it dynamically.